doc/docs/launch-checklist.md

```diff
@@ -7,13 +7,32 @@ Before you launch with Replicache in your product, it's a good idea to double-ch
 
 ## JS SDK
 
-- If you wish to change the type of a mutator (eg, the number or type of its arguments) you must choose a new name; Replicache does not handle mutator versioning.
-- At some point you will almost certainly wish to change the schema of mutations included in the `PushRequest` and the `clientView` returned in the `PullResponse`. The `ReplicacheOptions.schemaVersion` exists to facilitate this; it can be set by your app and is passed in both the `PushRequest` and `PullRequest`. Consider setting the `schemaVersion` from the start so that you don't later have to special case the "no schemaVersion" case.
+- If you wish to change the type of a mutator (eg, the number or type of its
```

Comment:
```suggestion
- If you wish to change the type of a mutator (eg, the number or type of its
```
```suggestion
- If you wish to change the signature of a mutator (eg, the number or type of its
```

I think maybe "signature" is a more commonly understood term, though maybe I'm just old school.